### PR TITLE
Linux default escaping

### LINKED ISSUE #12: Mentioning the escaping of !default in Linux for cslang-builder
I was trying to run the cslang-builder on linux with a test suite like:
`<path>/cslang-builder -ts !default,rft`
and received:
`-bash: !default,rft: event not found`

I felt that it was worth documenting that on linux the "!" should be escaped for it to work since the error message is not clear enough in my opinion.

@SamMarkowitz what do you think?

### PR DESCRIPTION
**Pull request by SamMarkowitz**
fixes #12 

Signed-off-by: Sam Markowitz sam.markowitz@hp.com

### CODE REVIEW

File: docs/developer_cloudslang.md [hunk 1]
```diff
@@ -9,13 +9,13 @@ CloudSlang content can be run from inside an existing Java application using Mav
 <dependency>
   <groupId>io.cloudslang</groupId>
   <artifactId>score-all</artifactId>
-  <version>0.1.271</version>
```
Comment (meirwah) [inline docs/developer_cloudslang.md:12]: how is this related to Linux default escaping??

**Comment by orius123** [comment]
I think it is a bash problem don't know about others (zsh..)

**Comment by meirwah** [comment]
@tudorlesan please review...

**Comment by tudorlesan** [comment]
:+1: seems fine to me

**Comment by oritstone** [comment]
Was is merged only to the new docs version? 
I think we want it also for 0.8 docs
